utils-intervalrdd/src/main/scala/org/bdgenomics/utils/intervalrdd/IntervalRDD.scala

```diff
@@ -63,10 +63,34 @@ class IntervalRDD[K <: Interval: ClassTag, V: ClassTag](
 
   /** The number of elements in the RDD. */
   override def count: Long = {
-    partitionsRDD.map(_.getTree.size).reduce(_ + _)
+    partitionsRDD.map(_.getTree.array.length).reduce(_ + _)
```

Comment:
Might be a bit cleaner to just add a `size` method on Forest, because I believe that'll be generally useful.